roadmap/implementors-guide/src/node/backing/pov-distribution.md

```diff
@@ -4,10 +4,118 @@ This subsystem is responsible for distributing PoV blocks. For now, unified with
 
 ## Protocol
 
-Handle requests for PoV block by candidate hash and relay-parent.
+`ProtocolId`: `b"povd"`
```

Comment:
Just to make sure, are we attaching a version to these protocol ids somewhere? I.e. something along the lines of `b"povd/1"`?

That would help tremendously evolving the protocol later in the future.

Reply:
These are attached to the network bridge. I figure these can handle versioning internally since the network bridge protocol is so generic

Reply:
I see. The network bridge would treats the protocol ids as opaque strings. Thereby later a subsystem could even just listen for both `povd` and `povd/2` during the transition period, correct?